### PR TITLE
ADD: Display gp floating clinics (gp-f)

### PR DESCRIPTION
**Pull request by bananica**
PixelSnap 2023-03-28 at 19 52 59
PixelSnap 2023-03-28 at 19 53 17

### CODE REVIEW

File: src/locales/it.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
     },
     "pediatrician": "pediatra",
     "youth": "giovani",
+    "floatingClinic": "Ambulanta za nepredeljene",
```
Comment (stefanb) [inline src/locales/it.json:26]:
```suggestion
    "floatingClinic": "Ambulanta za neopredeljene",
```

Comment (stefanb) [reply src/locales/it.json:26]:
```suggestion
    "floatingClinic": "Clinica per pazienti senza un medico",
```

V Izoli pa uporabljajo izraz "AMBULATORIO GENERALE PER GLI ASSICURATI SENZA MEDICO DI BASE"
https://www.zd-izola.si/ambulante-in-sluzbe/ambulanta-za-neopredeljene/ambulanta-za-neopredeljene

File: src/locales/en.json [hunk 2]
```diff
@@ -18,6 +18,7 @@
     "adults": "adults",
     "students": "students",
     "youth": "youth",
+    "floatingClinic": "Clinica per pazienti senza un medico personale",
```
Comment (stefanb) [inline src/locales/en.json:21]: Treba prevesti v angleščino

**Comment by stefanb** [comment]
Feedback form has visible field "obseg zaposlitve":
image
(gray background on the screenshot because of mouse hover)

File: src/locales/it.json (hunk 1 above)
Comment (stefanb) [reply src/locales/it.json:26]: @b100w11 nam lahko pomagaš s tem prevodom?

Comment (stefanb) [reply src/locales/it.json:26]:
```suggestion
    "floatingClinic": "Ambulatorio per gli persone senza medico",
```

Comment (stefanb) [reply src/locales/it.json:26]:
```suggestion
    "floatingClinic": "Ambulatorio per gli pazienti senza medico",
```

File: src/locales/en.json (hunk 2 above)
Comment (stefanb) [reply src/locales/en.json:21]:
```suggestion
    "floatingClinic": "Infirmary for patients without doctors",
```

File: src/locales/it.json [hunk 3]
```diff
@@ -23,6 +23,7 @@
     },
     "pediatrician": "pediatra",
     "youth": "giovani",
+    "floatingClinic": "Ambulatorio per gli pazienti senza medico",
```
Comment (stefanb) [inline src/locales/it.json:26]:
```suggestion
    "floatingClinic": "Ambulatorio per pazienti senza medico",
```

**Review by jalezi — COMMENTED** [review]
Have you tried with #008cd0? It is the icon's fill color.

**Comment by stefanb** [comment]
> Have you tried with #008cd0? It is the icon's fill color.

Indeed, that one looks even better. 
image
I didn't see it because it was not present in CSS, but set directly in SVG

**Review by jalezi — APPROVED** [review]
👍

**Review by stefanb — APPROVED** [review]

**Pull request merged**